woods_hole_obs_data/project_metadata.csv

```diff
@@ -54,3 +54,4 @@ project_name,contributor_name,project_title,project_summary,catalog_xml
 "RCNWR","N. Ganju","Rachel Carson NWR 2013","Oceanographic and water-quality observations made at 3 sites in the wetlands of the Rachel Carson National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/RCNWR/catalog.xml"
 "BW2011","N. Ganju"," Blackwater 2011"," Oceanographic and Water-Quality Measurements made at several sites in 2 watersheds in Blackwater National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BW2011/catalog.xml"
 "BARNEGAT_2014","N. Ganju"," Estuarine Physical Response to Storms (EPR), Barnegat Bay New Jersey, 2014","Oceanographic, and water-quality observations made at two locations in Barnegat Bay, NJ, 2014 to characterize the processes that affect sediment transport to tidal wetlands.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BARNEGAT_2014/catalog.xml"
+"CALCSTWET","N. Ganju",California Coastal Wetlands, 2013-2014","Oceanographic and Water Quality Measurements in two Southern California Coastal Wetlands, 2013-2014","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/CALCSTWET/catalog.html"
```

Comment:
Do you also want to add a `.` at the end of the `project_summary` so it looks like the others?